### ISSUE TITLE
t/module.t fails and hangs on FreeBSD systems

### ISSUE THREAD

**Issue by eserte** [gryphonshafer/Daemon-Device#2]
On my FreeBSD smokers (various OS versions, from 9 to 13) the test suite hangs after a failure in t/module.t:
```
#   Failed test 'Event actions appear to have all been conducted (and no extra actions)'
#   at t/module.t line 108.
# Compared array length of $data
#    got : array with 26 element(s)
# expect : array with 20 element(s)
# Looks like you failed 1 test of 8.
```

**Comment by gryphonshafer** [comment]
I'm able to intermittently reproduce this on Debian 4.9.

**Comment by gryphonshafer** [comment]
Resolved with 185d188, which will be released in v1.08 in a few minutes.
